daemon/cmd/main.go

```diff
@@ -135,6 +146,59 @@ func main() {
 				Name:  "swap",
 				Usage: "Swap operations",
 				Commands: []*cli.Command{
+					{
+						Name:  "in",
+						Usage: "Perform a swap in",
+						Flags: []cli.Flag{
+							&cli.StringFlag{
+								Name:     "payreq",
+								Usage:    "The Lightning invoice where the swap will be paid to",
+								Aliases:  []string{"p"},
+								Required: true,
+							},
+							&grpcPort,
+							&regtest,
+							&testnet,
```

Comment:
i'm thinking about this, what if we set this directly to the server and then take the network variable directly from it?

Reply:
that makes sense. The backend has a configuration endpoint that tells you the network, so no need to even set it in the daemon either